content/en/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -59,6 +59,12 @@ Run the following command to apply virtual services that will route all traffic
 $ kubectl apply -f @samples/bookinfo/networking/virtual-service-all-v1.yaml@
 {{< /text >}}
 
+And to apply `DestinationRule` items run following command.
```

Comment:
This isn't needed because the warning on line 46 tells the user to make sure it's already done.
```
If you haven't already, follow the instructions in [define the service versions](/docs/examples/bookinfo/#define-the-service-versions).
```

Reply:
ahh, I see. It is kind of hidden.

I executed command from section https://istio.io/latest/docs/setup/getting-started/#bookinfo
![image](https://github.com/istio/istio.io/assets/457542/ea5c3107-e95c-4aab-afcd-c6a9a368aaa5)

And did not clicked to link (because it looks like that it is not necessary) that goes to https://istio.io/latest/docs/examples/bookinfo/ ... let me update the text, to make it more explicit.

Reply:
I have force pushed changed, what do you think?